phoenix-core/src/it/java/org/apache/phoenix/end2end/ParallelStatsEnabledIT.java

```diff
@@ -38,7 +38,7 @@
 public abstract class ParallelStatsEnabledIT extends BaseTest {
 
     @BeforeClass
-    public static final void doSetup() throws Exception {
```

Comment:
Why did you remove final? I didn't see you re-implement this function in ExplainPlanWithStatsEnabledIT.java.